valhalla/jawn/src/lib/stores/experimentStore.ts

```diff
@@ -21,6 +21,11 @@ export interface RequestObj {
   provider: string;
 }
 
+export interface Score {
+  valueType: string;
```

Comment:
nit: Can we define the specific strings allowed here? 

ex: "number" | "string"

Reply:
We can't because we are mapping this interface to DB schema and valueType is a string